demo/src/network-score-test.ts

```diff
@@ -266,46 +266,35 @@ async function main() {
 
   console.log(`\n💠  Revised Rating - Credit Entry `)
 
-  let revisedRatingContent: Cord.IRatingContent = {
+  let revisedRatingContent = {
     ...ratingContent,
+    providerDid: transformedEntry.entry.providerDid,
     referenceId: revokedRatingUri,
     countOfTxn: 80,
-    totalRating: 280,
+    totalEncodedRating: 280,
```

Comment:
User sends the `totalRating`, and not `totalEncodedRating`.

Reply:
This has been addressed in the commit [6013dec](https://github.com/dhiway/cord.js/pull/190/commits/6013dec98ad58dd6ee331ad86d34e916b4b975d5)